src/Snapshot.php

```diff
@@ -61,6 +61,103 @@ public function load(string $connectionName = null)
         event(new LoadedSnapshot($this));
     }
 
+    public function loadStream(string $connectionName = null)
+    {
+        event(new LoadingSnapshot($this));
+
+        if ($connectionName !== null) {
+            DB::setDefaultConnection($connectionName);
+        }
+
+        $this->dropAllCurrentTables();
+
+        if ($this->compressionExtension === 'gz') {
+            $dumpFilePath = $this->decompress();
+        } else {
+            $dumpFilePath = $this->disk->path($this->fileName);
+        }
+
+        $this->streamFileIntoDB($dumpFilePath, $connectionName);
+
+        event(new LoadedSnapshot($this));
+    }
+
+    public function streamFileIntoDB($path, string $connectionName = null)
+    {
+        if ($connectionName !== null) {
+            DB::setDefaultConnection($connectionName);
+        }
+
+        $tmpLine = '';
+        $lines = file($path);
+        $errors = [];
+
+        foreach ($lines as $line) {
+
+            // Skip it if line is a comment
+            if (substr($line, 0, 2) == '--' || trim($line) == '') {
+                continue;
+            }
+
+            // Add this line to the current segment
+            $tmpLine .= $line;
+
+            // If the line ends with a semicolon, it is the end of the query - run it
+            if (substr(trim($line), -1, 1) == ';') {
+                try {
+                    DB::connection($connectionName)->unprepared($tmpLine);
+                } catch (\Exception $e) {
```

Comment:
Import this so we can avoid that `\`

Reply:
PHPStorm give me warnings about this without the `\` even if I import the exception.

Reply:
PHPStorm does not give warning for this if importing the proper exception as the top like so:

```php
use Exception;
...
try {
    DB::connection($connectionName)->unprepared($tmpLine);
} catch (Exception $e) {
...
}